core/extraction/utils.py

```diff
@@ -98,3 +99,7 @@ def extract_postcodes(s: str | float) -> list[str]:
     else:
         postcode_area_matches = re.findall(POSTCODE_REGEX, str(s))
     return postcode_area_matches
+
+
+def join_as_string(s: Sequence) -> str:
```

Comment:
can we try and avoid using single letter var names where possible (I know there's a few throughout the Extraction code) but we can try and not have any moving forwards. Ta

Reply:
also, please add a quick docstring